src/olympia/pages/templates/pages/shield_study_9.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Get Synced</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_9/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Get Synced</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/get-synced-v1-0/platform:3/addon-760109-latest.xpi?src=shield-study-9"><button class="button primary large">Get It Now</button></a>
+        </div>
+        <div class="content"><p>Mozilla wants to know if we can improve your experience when linking and transferring content between desktop and mobile devices. Participating in this study will help us find out!</p>
+<h2 id="what-will-happen-next">What will happen next</h2>
+<p><strong>If you agree</strong> to participate:</p>
+<ol>
+<li>We will add additional functionality to Firefox notifications.</li>
+<li>Continue using Firefox as you normally would. You may not notice any changes immediately.</li>
+<li>After 2 weeks, the study will end on its own. Your Firefox will go back to normal.</li>
+<li>We will ask you to fill out a quick optional survey to tell us about your experience. The survey should take less than 5 minutes to complete.</li>
+</ol>
+<h2 id="leaving-the-study">Leaving the study</h2>
+<p>The study will expire on its own in 14 days.  You may leave the study early.  To do so, follow these steps:</p>
```

Comment:
Oh, that is a fussy one :)  I will fix it when I fix the url above :)  +1  .

(The dangers of copying out of google docs